docs/page-objects/rules.rst

```diff
@@ -5,8 +5,17 @@ Rules
 =====
 
 Rules are :class:`~.ApplyRule` objects that tell web-poet which :ref:`page
-object class <page-object-classes>` to use for a given combination of input URL
-and :ref:`item type <item-types>`.
+object class <page-object-classes>` to use based on user input, i.e. the target
+URL and an output class (a :ref:`page object class <page-object-classes>` or an
+:ref:`item class <items>`).
+
+Rules are necessary to enable using an item class as output class, and can also
```

Comment:
Hm, I'm not sure it's clear here what "enable using an item class as output class" means

Reply:
https://github.com/scrapinghub/web-poet/pull/169/commits/7b9c39a439cbdff5eefa164bead0ba5e22fb1310 ?